include/zephyr/bluetooth/iso.h

```diff
@@ -110,11 +110,14 @@ enum bt_iso_state {
 };
 
 
+/**
+ * @brief ISO Channel Type.
+ */
 enum bt_iso_chan_type {
-	BT_ISO_CHAN_TYPE_NONE,
-	BT_ISO_CHAN_TYPE_CONNECTED,
-	BT_ISO_CHAN_TYPE_BROADCASTER,
-	BT_ISO_CHAN_TYPE_SYNC_RECEIVER
+	BT_ISO_CHAN_TYPE_NONE,		/**< No channel type */
+	BT_ISO_CHAN_TYPE_CONNECTED,	/**< Connected */
+	BT_ISO_CHAN_TYPE_BROADCASTER,	/**< Isochronous */
```

Comment:
```suggestion
	BT_ISO_CHAN_TYPE_BROADCASTER,	/**< Isochronous Broadcaster */
```

Reply:
of course, totally a typo on my end! Thanks for paying attention :)